ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -2921,7 +2922,13 @@ LdrpInitSecurityCookie(PLDR_DATA_TABLE_ENTRY LdrEntry)
             {
                 NewCookie = DEFAULT_SECURITY_COOKIE + 1;
             }
-
+#ifdef _WIN64
+            /* Some images expect first 16 bits to be kept clean (like in default cookie) */
```

Comment:
It would be of interest to tell us which images these are (their file names)

Reply:
wpprecorder.sys is one that I know.